README.md

```diff
@@ -24,16 +24,19 @@ Or install it yourself as:
 
     require 'lightstep'
 
-    # Initialize the singleton tracer
-    LightStep.configure(component_name: 'lightstep/ruby/example', access_token: 'your_access_token')
+    # Initialize the OpenTracing global tracer
+    Opentracing.global_tracer = LightStep::Tracer.new(
```

Comment:
yep, thanks! Wish ruby had the go-style "executable examples" so this was more guaranteed to work :)